packages/wdio-config/src/lib/ConfigParser.ts

```diff
@@ -165,22 +174,23 @@ export default class ConfigParser {
     /**
      * get excluded files from config pattern
      */
-    getSpecs (capSpecs, capExclude) {
-        let specs = ConfigParser.getFilePaths(this._config.specs)
+    getSpecs (capSpecs: string[], capExclude: string[]) {
+        let specs = ConfigParser.getFilePaths(this._config.specs!)
         let spec  = Array.isArray(this._config.spec) ? this._config.spec : []
-        let exclude = ConfigParser.getFilePaths(this._config.exclude)
+        let exclude = ConfigParser.getFilePaths(this._config.exclude!)
         let suites = Array.isArray(this._config.suite) ? this._config.suite : []
 
         /**
          * check if user has specified a specific suites to run
          */
         if (suites.length > 0) {
-            let suiteSpecs = []
+            let suiteSpecs: string[] = []
             for (let suiteName of suites) {
-                // TODO: log warning if suite was not found
-                let suite = this._config.suites[suiteName]
-
-                if (suite && Array.isArray(suite)) {
+                let suite = this._config.suites?.[suiteName]
+                if (!suite) {
+                    console.warn(`No suite was found with name "${suiteName}"`)
```

Comment:
Let's use WebdriverIOs logger utility for this
```suggestion
                    log.warn(`No suite was found with name "${suiteName}"`)
```

Reply:
Ok I'll update this.